znai-reactjs/src/doc-elements/Documentation.js

```diff
@@ -249,6 +249,10 @@ export class Documentation extends Component {
             this.setState({mode: DocumentationModes.PRESENTATION})
         } else if (mode === DocumentationModes.DEFAULT && e.code === 'KeyP' && e.altKey) {
             this.setState({mode: DocumentationModes.PRINT})
+        } else if (mode === DocumentationModes.DEFAULT && e.code === 'ArrowLeft') {
```

Comment:
Arrow keys without any modification may not be ideal. Do you see this behavior on some of the documentation websites?
Default arrow keys behavior is scroll left and right. Changing defaults could frustrate users.

Reply:
For znai, a left or a right keys are very natural for me.
Do you have any recommended keys then?
I initially used alt + left / alt + right, then I found that Chrome has default key setups for them.

Reply:
How about H/L like vi?

Reply:
Do we have users requesting it? Left right is natural when the pages are shorter than screen. 
I am trying to better understand the use case. Is it for quickly moving through the docs?

Reply:
Yes, I got a request from a user that he wants shortcuts to navigate pages.

Reply:
How about Ctrl + Arrow? We probably should add a shortcut captions to the buttons (I think we used to have one for search)

Reply:
Chrome already defined them.


Move your cursor to the beginning of the previous word in a text field | Ctrl + Left arrow

Move your cursor to the next word | Ctrl + Right arrow

Reply:
If we talk about text field then Left Right is 100% no go as they move your cursor. 
We also don’t have (at least now) any text fields in the docs.

Reply:
Why don't you like h/l as in vi?

Reply:
I kind of do, emacs people may not :) 
Jokes aside I actually added `/` for search, but nobody knows about that key. Also didn't want to override default `ctrl + F`.

`h`, `l` are left and right one char, not one page. But maybe semantics are not that important here.
I would still prefer in addition to have more conventional keys like arrows + modificator.

Ctrl + Left/Right should work.

Reply:
Sounds good.
I will update the change soon with Ctrl + Left/Right.

Reply:
could you also please update docs somewhere in the `flow` file or nearby about shortcuts for page navigation?

Reply:
Added shortcuts in flow.

Reply:
<img width="1124" alt="Screen Shot 2019-12-03 at 5 17 21 PM" src="https://user-images.githubusercontent.com/415416/70094702-1a412500-15f1-11ea-894d-766b5ffadace.png">

Reply:
nice!